web/lib/firebase/schema.ts

```diff
@@ -0,0 +1,19 @@
+import { db } from './init'
```

Comment:
Mixed feelings on a schema file, which would be another file to update instead of just the one for the collection.

I kind of understand the benefit of being able to see all the top-level collections at once. But also, this currently doesn't include the sub collections which makes it a bit misleading.

Reply:
I think including the subcollections would be totally reasonable, but I am not 100% confident whether it's better to expose them as a Firebase `collectionGroup`, a function from `parentId` to `collection`, or something else, so I decided I would just get your guys' feedback on this version first.